src/components/Facets.tsx

```diff
@@ -29,17 +38,21 @@ export interface FacetsProps {
  * and {@link HierarchicalFacets} components can be used instead for more control over facet
  * configuration.
  *
+ * To override a single facet, use {@link StandardFacet} and set the field id on overriddenFieldIds.
```

Comment:
So I don't think that's actually how we want this to work. I think we just want to base it on whether or not a child is provided with a field id prop that matches one of the field ids from the response.

Reply:
Done